lib/fluent/plugin/in_tail.rb

```diff
@@ -117,6 +114,30 @@ def configure_tag
       end
     end
 
+    def configure_encoding
+      if @encoding
+        @encoding = parse_encoding_param(@encoding)
+        if @from_encoding
+          @from_encoding = parse_encoding_param(@from_encoding)
+        end
+      else
+        if @from_encoding
+          $log.warn "'from_encoding' parameter must be specified in conjunction with 'encoding' paramter."
```

Comment:
Use `log` instead of `$log` to enable plugin logger (`log_level` for each plugin sections).
And I think it's ok to say just `must be specified with "encoding" parameter`.